Vagrantfile

```diff
@@ -184,7 +184,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       grf.vm.hostname = "#{LABEL_PREFIX}grafana#{i}"
       if ASSIGN_STATIC_IP
         grf.vm.network :private_network,
-          ip: "#{PUBLIC_SUBNET}.3#{i}"
+          ip: "#{PUBLIC_SUBNET}.4#{i}"
```

Comment:
`.4x` is also already used by clients nodes

```console
$ grep 'ip: "#{PUBLIC_SUBNET}\.' Vagrantfile 
          ip: "#{PUBLIC_SUBNET}.3#{i}"
          ip: "#{PUBLIC_SUBNET}.3#{i}"
          ip: "#{PUBLIC_SUBNET}.4#{i}"
          ip: "#{PUBLIC_SUBNET}.5#{i}"
          ip: "#{PUBLIC_SUBNET}.6#{i}"
          ip: "#{PUBLIC_SUBNET}.7#{i}"
          ip: "#{PUBLIC_SUBNET}.8#{i}"
          ip: "#{PUBLIC_SUBNET}.9#{i}"
          ip: "#{PUBLIC_SUBNET}.1#{i}"
          ip: "#{PUBLIC_SUBNET}.10#{i}"
```

I don't see any .2x so we could change to this. wdyt ?

Reply:
Sure,  I think `.2x` is fine.